apps/web/pages/buildersummer.tsx

```diff
@@ -12,9 +12,9 @@ export default function OnChainSummer() {
   return (
     <>
       <Head>
```

Comment:
Let's delete this head block entirely and make any desired edits here: https://github.com/base-org/web/blob/master/apps/web/pages/_document.tsx#L58